pandas/tests/frame/test_nonunique_indexes.py

```diff
@@ -416,9 +416,10 @@ def test_columns_with_dups(self):
             [[1, 2, 1., 2., 3., 'foo', 'bar']], columns=list('ABCDEFG'))
         assert_frame_equal(df, expected)
 
+        # TODO: fix this. does not raise.
         # this is an error because we cannot disambiguate the dup columns
-        pytest.raises(Exception, lambda x: DataFrame(
-            [[1, 2, 'foo', 'bar']], columns=['a', 'a', 'a', 'a']))
+        # with pytest.raises(Exception, match="<enter message here>"):
+        #     DataFrame([[1, 2, 'foo', 'bar']], columns=['a', 'a', 'a', 'a'])
```

Comment:
does not raise. added a TODO. not in this PR though.

Reply:
Was this not failing CI before?

Reply:
@WillAyd . case of lambda x: with no x raises TypeError and so passed CI.